applications/GeoMechanicsApplication/tests/test_parameter_field.py

```diff
@@ -71,21 +73,29 @@ def test_parameter_field_with_python_umat_parameters(self):
         try:
             shutil.copy(custom_python_file, new_custom_script_path)
         except shutil.SameFileError as e:
-             print(f"Source and destination represents the same file. {e}")
-             raise
+            print(f"Source and destination represents the same file. {e}")
+            raise
         except PermissionError as e:
-             print(f"Permission denied: {e}")
-             raise
+            print(f"Permission denied: {e}")
+            raise
         except IOError as e:
-             print(f"Try to change the destination file permission {e}")
-             os.chmod(new_custom_script_path, stat.S_IRWXU)
-             shutil.copy(custom_python_file, new_custom_script_path)
+            print(f"Try to change the destination file permission {e}")
+            os.chmod(new_custom_script_path, stat.S_IRWXU)
+            shutil.copy(custom_python_file, new_custom_script_path)
         except Exception as e:
-             print(f"Error occurred while copying the file {e}.")
-             raise
+            print(f"Error occurred while copying the file {e}.")
+            raise
+
```

Comment:
```suggestion
        except:
            print("an unknown error occurred")
            raise
```

or if you really wanna catch `BaseException`:
```suggestion
        except BaseException as e:
            raise RuntimeError("an unknown error occurred") from e
```

Reply:
done the last suggestion.